lib/helpers.js

```diff
@@ -22,6 +22,11 @@ export const isExisty = x => {
   return x != null; // eslint-disable-line eqeqeq
 };
 
+// Coerce a usable value or nothing at all
+export const existyValue = x => {
+  if (isExisty(x)) return x;
+};
```

Comment:
I'm wondering if it's worth it to walk every API response and just undef every empty object/string.